ingestors/kafka/src/main/scala/hydra/kafka/endpoints/BootstrapEndpointV2.scala

```diff
@@ -34,15 +31,16 @@ import scala.util.{Failure, Success}
 final class BootstrapEndpointV2[F[_]: Futurable](
     createTopicProgram: CreateTopicProgram[F],
     defaultTopicDetails: TopicDetails,
-    metadataAlgebra: MetadataAlgebra[F]
-) extends CorsSupport with TopicMetadataV2Adapter {
+) extends CorsSupport {
 
   import TopicMetadataV2Parser._
 
   val route: Route = cors(settings) {
-    pathPrefix("v2" / "streams") {
-      post {
-        pathEndOrSingleSlash {
+    pathPrefix("v2" / "topics") {
+      put {
+        extractUnmatchedPath { topic =>
+          topic.toString().replace("/","")
+          //TODO: Change the way creating a topic works
```

Comment:
Not sure what this is referring to, but can we break this out and add it as an issue on the repo instead?